source/renderer/app/components/voting/voting-info/Headline.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React from 'react';
+import BigNumber from 'bignumber.js';
+import { defineMessages, injectIntl } from 'react-intl';
+
+import { ExternalLinkButton } from '../../widgets/ExternalLinkButton';
+import type { Intl } from '../../../types/i18nTypes';
+
+import styles from './Headline.scss';
+
+const messages = defineMessages({
```

Comment:
We could consider moving these messages, and all for `voting-info` component into a dedicated messages file to make the components even smaller, but this just a matter of personal preference.

Reply:
Updated, thanks!